Oric-DFLAT/vdp/graph.s

```diff
@@ -195,7 +195,8 @@ gr_scrngeom_hires
 	db gr_hires_x, 0
 	db gr_hires_y, 0
 
-	; Only need to initialise text height, width is same as before
+	; Only need to initialise text height, width is same as before (iss: really? ;)
+	db gr_text_w, 40
```

Comment:
However a better fix is not to initialise this here with two extra bytes, but remove the '-1' that terminates the base table and I can also remove gr_mode initialisation here as it is done in gr_scrngeom_txt.